rocAL/rocAL/source/pipeline/image.cpp

```diff
@@ -81,6 +81,16 @@ const std::vector<uint32_t>& ImageInfo::get_roi_height_vec() const
     return *_roi_height;
 }
 
+const std::vector<uint32_t>& ImageInfo::get_original_roi_width_vec() const
```

Comment:
please remove word from here "roi"

Reply:
Changed the function name in the latest commit